update/ansible/playbook/tasks/roles/nginx/files/ssl/generate-ssl-certificate

```diff
@@ -3,6 +3,35 @@
 set -o errexit
 
 mkdir -p /srv/nginx
+if [ ! -w "/srv/nginx" ]; then
+    echo "Directory /srv/nginx is not writable";
+    if [ ! -e /srv/nginx/certificate.key ]
+    then
+        echo "The file /srv/nginx/certificate.key does not exist. Exiting...";
+        exit 1;
+    fi
+
+    if [ ! -e /srv/nginx/certificate.crt ]
+    then
+        echo "The file /srv/nginx/certificate.crt does not exist. Exiting...";
+        exit 1;
+    fi
+
+    if [ ! -e /srv/nginx/ca-certs.pem ]
+    then
+        echo "The file /srv/nginx/ca-certs.pem does not exist. Exiting...";
+        exit 1;
+    fi
+
+    if [ ! -e /srv/nginx/dhparam.pem ]
+    then
+        echo "The file /srv/nginx/dhparam.pem does not exist. Exiting...";
+        exit 1;
+    fi
+
+    echo "Using existing self-signed certificate";
+    exit 0;
+fi
```

Comment:
```suggestion
if [ ! -w "/srv/nginx" ]; then
    echo "Directory /srv/nginx is not writable";
    SHOULD_EXIT=0

    if [ ! -e /srv/nginx/certificate.key ]; then
        echo "The file /srv/nginx/certificate.key does not exist.";
        SHOULD_EXIT=1;
    fi

    if [ ! -e /srv/nginx/certificate.crt ]; then
        echo "The file /srv/nginx/certificate.crt does not exist.";
        SHOULD_EXIT=1;
    fi

    if [ ! -e /srv/nginx/ca-certs.pem ]; then
        echo "The file /srv/nginx/ca-certs.pem does not exist.";
        SHOULD_EXIT=1;
    fi

    if [ ! -e /srv/nginx/dhparam.pem ]; then
        echo "The file /srv/nginx/dhparam.pem does not exist.";
        SHOULD_EXIT=1;
    fi

    if [ "$SHOULD_EXIT" = 1 ]; then
        echo "FATAL: one or more files required for nginx could not be found. Exiting..." >&2
        exit 1
    fi

    echo "Using existing self-signed certificate";
    exit 0;
fi
```

Reply:
That will tell the user about all the missing files at once, not one-by-one :)

Reply:
Done